### DISCUSSION TITLE
Building - Latest version, BOM

### DISCUSSION THREAD

**Discussion by anm** [peterantypas/maiana#143] [General]
Hi,

I'd like to build a few MAIANAs. I just wanted to check what version I should be building. It looks like Board-11.9 is the latest - is it complete and tested?

Does someone have a complete BOM with order numbers? In kicad I don't see specs like tolerances / voltages for the passives.

There seem to be a few variants of the Si4660, with Si4X6X-B1B being replaced with SI4X6X-C2A. It looks like the new part is better, except for the critical flaw that it doesn't cover AIS frequencies. B1B is now NRND, that's not great.

https://www.silabs.com/documents/public/application-notes/AN897.pdf

Thanks,
River

**Comment by anm** [comment]
Ah, also, what about PCB stackup / impedance control?

I'll try and get JLCPCB to build some, unless you have designed for some board they don't have you get elsewhere?

Edit: I see you specced JLC2313.

**Comment by unkn0wn107** [comment]
Hello, have you built your own ?

Discussing about it here : [Talking about going back to production](https://github.com/peterantypas/maiana/discussions/148)

**Comment by ra1amx** [comment]
> Hi,
> 
> I'd like to build a few MAIANAs. I just wanted to check what version I should be building. It looks like Board-11.9 is the latest - is it complete and tested?
> 
> Does someone have a complete BOM with order numbers? In kicad I don't see specs like tolerances / voltages for the passives.
> 
> There seem to be a few variants of the Si4660, with Si4X6X-B1B being replaced with SI4X6X-C2A. It looks like the new part is better, except for the critical flaw that it doesn't cover AIS frequencies. B1B is now NRND, that's not great.
> 
> https://www.silabs.com/documents/public/application-notes/AN897.pdf
> 
> Thanks, River

Hi. Also interesting BOM. On mu side I will made with local modules (Milandr)

**Comment by matthew-quirke** [comment]
Hi Ivan (ra1amx) i saw you comment below and link about the SI4X6X-C2A revision NOT supporting the AIS channels 160.975 and 161.025

"with SI4X6X-C2A. It looks like the new part is better, except for the critical flaw that it doesn't cover AIS frequencies. B1B is now NRND, that's not great"

https://www.silabs.com/documents/public/application-notes/AN897.pdf_

In the reference link I didn't find any mention of the AIS channels Not being supported - Did i miss it?

Si4460-C2A | 142–1050 MHz
Si4461-C2A | 142–1050 MHz
Si4463-C2A | 142–1050 MHz
Si4467-A2A | 119–1050 MHz
Si4468-A2A | 119–1050 MHz 

@anm - Also interested in which tolerances like the dielectric for the capacitors - 

It looks like Peter used C0G, NP0 (light grey) for RF capacitors (@ 25V or 50V to cover mismatch?)  and  X7R (light brown ) OR (Y5V) dark brown for power generic decoupling etc?
